app/controllers/compute_resources_vms_controller.rb

```diff
@@ -21,7 +21,7 @@ def index
       format.html
       format.json do
         if @compute_resource.supports_vms_pagination?
-          render :partial => "compute_resources_vms/index/#{@compute_resource.provider.downcase}.json"
+          render :partial => "compute_resources_vms/index/#{@compute_resource.provider.downcase}_json"
```

Comment:
I guess this also will do, but without a need to rename the file itself:

```suggestion
          render :partial => "compute_resources_vms/index/#{@compute_resource.provider.downcase}, formats: :json"
```

Reply:
IIRC this was the first thing I've tried an it did not work... However, I've tested this again today and it seems to work perfectly fine. :man_shrugging: 

I've fixed the PR accordingly.